docs/developer/thumbnails.mdx

```diff
@@ -49,6 +49,11 @@ given size and format for the first time, a proxy URL is returned. When the prox
 is accessed, the requested thumbnail is created and the client is redirected.
 Future requests return the thumbnail's URL directly.
 
+:::note
+Proxy URL will be constructed using a domain that Saleor is set to run on. This can be changed using
```

Comment:
it is mentioned above the added note

Reply:
Not sure if this belongs here. Shouldn't there be another section of the docs explaining how URLs are constructed?

Reply:
@patrys If I were having problems with generated thumbnail URLs that would be the place I would look for the info on that. Or maybe I misunderstand and by another section, you mean that it should not be `note` but a new paragraph/section of the same document in the same place?

Reply:
IMO it should link to documentation explaining how URLs are constructed instead of copying that information (and details on how to change it) to all the relevant places.

Reply:
@patrys separate issue created for additional documentation https://github.com/saleor/saleor-docs/issues/845